### PR TITLE
🎨 clean up Oracle Modules

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

File: src/oracle-modules/AbstractOracleModule.sol [hunk 1]
```diff
@@ -89,11 +84,18 @@ abstract contract OracleModule is Owned {
...
-     * @dev The oracle rate expresses how much units of the QuoteAsset (18 decimals precision) are required
-     * to buy 1 unit of the BaseAsset.
+     * @return oracleRate The rate of the BaseAsset in units of QuoteAsset, with 18 decimals precision.
+     * @dev The oracle rate expresses how much units of the QuoteAsset are required
+     * to buy 1 unit of the BaseAsset, with 18 decimals precision.
+     * Example: If you have an oracle (WBTC/USDC).
+     *  - The BaseAsset is Wrapped Bitcoin (WBTC), which has 8 decimals.
+     *  - The QuoteAsset is USDC, which has 6 decimals.
+     *  - Assume an exchange rate from Bitcoin to USD of $30 000.
+     *  -> You need $30 000 (or 30 000 * 10^6 USDC) to buy 1 Bitcoin (or 1 * 10^8 WBTC).
```
Comment (j-vp) [inline src/oracle-modules/AbstractOracleModule.sol:96]: 10**n for exponents

File: src/oracle-modules/AbstractOracleModule.sol [hunk 2]
```diff
@@ -89,11 +84,18 @@ abstract contract OracleModule is Owned {
...
+     *  -> You need 300 units of USDC to buy one unit of WBT.
+     * Since we use 18 decimals precision, the oracleRate will be 300 * 10^18.
```
Comment (j-vp) [inline src/oracle-modules/AbstractOracleModule.sol:98]: 10**n for exponents

File: src/oracle-modules/ChainlinkOracleModule.sol [hunk 3]
```diff
@@ -136,19 +138,27 @@ contract ChainlinkOracleModule is OracleModule {
      * @return oracleRate The rate of the BaseAsset in units of QuoteAsset, with 18 Decimals precision.
      * @dev The oracle rate reflects how much units of the QuoteAsset (18 decimals precision) are required,
      * to buy 1 unit of the BaseAsset.
+     * Example: If you have an oracle (WBTC/USDC).
+     *  - The BaseAsset is Wrapped Bitcoin (WBTC), which has 8 decimals.
+     *  - The QuoteAsset is USDC, which has 6 decimals.
+     *  - Assume an exchange rate from Bitcoin to USD of $30 000.
+     *  -> You need $30 000 (or 30 000 * 10^6 USDC) to buy 1 Bitcoin (or 1 * 10^8 WBTC).
```
Comment (j-vp) [inline src/oracle-modules/ChainlinkOracleModule.sol:145]: 10**n for exponents

File: src/oracle-modules/ChainlinkOracleModule.sol [hunk 4]
```diff
@@ -136,19 +138,27 @@ contract ChainlinkOracleModule is OracleModule {
...
+     *  -> You need 300 units of USDC to buy one unit of WBT.
+     * Since we use 18 decimals precision, the oracleRate will be 300 * 10^18.
```
Comment (j-vp) [inline src/oracle-modules/ChainlinkOracleModule.sol:147]: 10**n for exponents

**Review by zekiblue — APPROVED** [review]
lgtm

**Pull request merged**